code/game/objects/objs.dm

```diff
@@ -416,7 +416,7 @@
 
 	var/offset_x = worn_x_dimension
 	var/offset_y = worn_y_dimension
-	if(inhands)
+	if(inhands == 1 || inhands == 0)
```

Comment:
what is the purpose of this

Reply:
the 64x64 right inhands are broken because if(ifhands) is only valid for the value 1 and having an item in your right hand sends the inhands variable to 0

Reply:
There is no point to this if check then. It can only ever possibly be 1 or 0